src/main/resources/templates/db_scripts/schema.json

```diff
@@ -50,11 +100,156 @@
           "fieldName": "purchaseOrderId",
           "targetTable": "purchase_order"
         }
+      ],
+      "index": [
+        {
+          "fieldName": "eresource.accessProvider",
+          "caseSensitive": false
+        },
+        {
+          "fieldName": "eresource.activated",
+          "caseSensitive": false
+        },
+        {
+          "fieldName": "eresource.expectedActivation",
+          "caseSensitive": false
+        },
+        {
+          "fieldName": "eresource.trial",
+          "caseSensitive": false
+        },
+        {
+          "fieldName": "eresource.materialType",
+          "caseSensitive": false
+        },
+        {
+          "fieldName": "details.subscriptionFrom",
+          "caseSensitive": false
+        },
+        {
+          "fieldName": "details.subscriptionTo",
+          "caseSensitive": false
+        },
+        {
+          "fieldName": "location.locationId",
+          "caseSensitive": false
+        },
+        {
+          "fieldName": "physical.receiptDue",
+          "caseSensitive": false
+        },
+        {
+          "fieldName": "physical.expectedReceiptDate",
+          "caseSensitive": false
+        },
+        {
+          "fieldName": "physical.volumes",
+          "caseSensitive": false
+        },
+        {
+          "fieldName": "physical.materialType",
+          "caseSensitive": false
+        },
+        {
+          "fieldName": "claim.grace",
+          "caseSensitive": false
+        },
+        {
+          "fieldName": "claim.sent",
+          "caseSensitive": false
+        },
+        {
+          "fieldName": "collection",
+          "caseSensitive": false
+        },
+        {
+          "fieldName": "receiptStatus",
+          "caseSensitive": false
+        },
+        {
+          "fieldName": "paymentStatus",
+          "caseSensitive": false
+        },
+        {
+          "fieldName": "orderFormat",
+          "caseSensitive": false
+        },
+        {
+          "fieldName": "receiptDate",
+          "caseSensitive": false
+        },
+        {
+          "fieldName": "metadata.createdDate",
+          "caseSensitive": false
+        },
+        {
+          "fieldName": "rush",
+          "caseSensitive": false
+        },
+        {
+          "fieldName": "source",
+          "caseSensitive": false
+        }
+      ],
+      "fullTextIndex": [
+        {
+          "fieldName": "publisher"
+        },
+        {
+          "fieldName": "donor"
+        },
+        {
+          "fieldName": "requester"
+        },
+        {
+          "fieldName": "selector"
+        }
+      ],
+      "ginIndex": [
```

Comment:
ginIndexes are used for regex queries, from the UI we would be searching for poLineNumber and requestor, would be better to add those too